blocks/fragment-tabs/fragment-tabs.js

```diff
@@ -119,18 +128,10 @@ export default async function decorate(block) {
     <div id="tab-content" class="tabs-content"></div>
   `;
 
+  // await tabs.preloadTabs();
```

Comment:
Seems like an earlier attempt.  If so, should probably remove this line to clean up the code.